[2024-11-25T07:05:56.981+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-25T07:05:56.989+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Citibike_Data_ETL.load_citibike_data manual__2024-11-25T07:04:21.460384+00:00 [queued]>
[2024-11-25T07:05:56.993+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Citibike_Data_ETL.load_citibike_data manual__2024-11-25T07:04:21.460384+00:00 [queued]>
[2024-11-25T07:05:56.993+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-11-25T07:05:56.998+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_citibike_data> on 2024-11-25 07:04:21.460384+00:00
[2024-11-25T07:05:57.004+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=8314) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-25T07:05:57.004+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Citibike_Data_ETL', 'load_citibike_data', 'manual__2024-11-25T07:04:21.460384+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/ETL_citibike.py', '--cfg-path', '/tmp/tmpa4m40jpd']
[2024-11-25T07:05:57.005+0000] {standard_task_runner.py:91} INFO - Job 81: Subtask load_citibike_data
[2024-11-25T07:05:57.005+0000] {standard_task_runner.py:63} INFO - Started process 8317 to run task
[2024-11-25T07:05:57.032+0000] {task_command.py:426} INFO - Running <TaskInstance: Citibike_Data_ETL.load_citibike_data manual__2024-11-25T07:04:21.460384+00:00 [running]> on host 5de3b6ca18a7
[2024-11-25T07:05:57.074+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='shatayu.thakur@sjsu.edu' AIRFLOW_CTX_DAG_OWNER='Shatayu' AIRFLOW_CTX_DAG_ID='Citibike_Data_ETL' AIRFLOW_CTX_TASK_ID='load_citibike_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-25T07:04:21.460384+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-25T07:04:21.460384+00:00'
[2024-11-25T07:05:57.074+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-25T07:05:57.081+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-11-25T07:05:57.081+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.11-linuxkit-aarch64-with-glibc2.36
[2024-11-25T07:05:57.082+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-25T07:05:57.702+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-25T07:06:03.187+0000] {logging_mixin.py:188} INFO - Row count for data to be pushed:  2982715
[2024-11-25T07:06:03.385+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-25T07:06:03.386+0000] {logging_mixin.py:188} INFO - Stage created successfully.
[2024-11-25T07:06:03.791+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-11-25T07:06:03.792+0000] {logging_mixin.py:188} INFO - An error occurred during stock data load: 090106 (22000): 01b89a2a-0004-386b-0002-55f300042b5a: Cannot perform STAGE PUT. This session does not have a current schema. Call 'USE SCHEMA', or use a qualified name.
[2024-11-25T07:06:03.792+0000] {connection.py:762} INFO - closed
[2024-11-25T07:06:03.836+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2024-11-25T07:06:04.051+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-25T07:06:04.052+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-25T07:06:04.059+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Citibike_Data_ETL, task_id=load_citibike_data, run_id=manual__2024-11-25T07:04:21.460384+00:00, execution_date=20241125T070421, start_date=20241125T070556, end_date=20241125T070604
[2024-11-25T07:06:04.103+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-25T07:06:04.117+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-25T07:06:04.119+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
